python/example/accuracy_pitfalls.ipynb

```diff
@@ -0,0 +1,338 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 47,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Accuracy: Pitfalls and Edge Cases\n",
+    "\n",
+    "This notebook describes OpenDP's accuracy calculations, and ways in which an analyst might be tripped up by them.\n",
+    "\n",
+    "### Overview\n",
+    "\n",
+    "#### Accuracy vs. Confidence Intervals\n",
+    "\n",
+    "Each privatizing mechanism (e.g. Laplace, Gaussian) in OpenDP has an associated accuracy that is a function of the requested privacy usage, sensitivity of the function whose values is being privatized, etc. Imagine you have data $D$ and you want, for some function $\\phi$ to return $\\phi(D)$ in a differentially private way -- we will call this value $\\phi_{dp}(D)$. An $\\alpha$-level accuracy guarantee $a$ promises that, over infinite runs of the privatizing mechanism on the data in question,\n",
```

Comment:
I know this came from before, but now it's just a function of alpha, and either sigma or accuracy.